esb-connector-creatio-crm/src/test/java/org/wso2/carbon/connector/integration/test/creatiocrm/CreatioCRMRestConnectorIntegrationTest.java

```diff
@@ -0,0 +1,261 @@
+        eiRequestHeadersMap.get(esbRestResponse.getHeadersMap().get("BPMCSRF"));
+        eiRequestHeadersMap.get(esbRestResponse.getHeadersMap().get("set-cookie"));
+        eiRequestHeadersMap.get(esbRestResponse.getHeadersMap().get("Cookie"));
+
+        methodName = "postCase";
+
+        esbRestResponse = sendJsonRestRequest(getProxyServiceURLHttp(methodName),
+                "POST", eiRequestHeadersMap, "esb_casePost_optional.json");
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 201);
+
+        String cId = esbRestResponse.getBody().getString("Id");
+
+        methodName = "getCase";
+        String endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "GET",
+                apiRequestHeadersMap);
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 200);
+
+        methodName = "updateCase";
+        endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "PUT",
+                apiRequestHeadersMap, "esb_caseUpdate_optional.json");
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 204);
+
+        methodName = "deleteCase";
+        endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "DELETE",
+                apiRequestHeadersMap);
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 204);
+    }
+
+    @Test(groups = {"wso2.ei"}, description = "Creatio Case integration test for CRUD API call without Auth data")
+    public void caseTestWithoutAuth() throws Exception {
+
+        String methodName = "postCase";
+
+        RestResponse<JSONObject> esbRestResponse = sendJsonRestRequest(getProxyServiceURLHttp(methodName),
+                "POST", eiRequestHeadersMap, "esb_casePost_optional.json");
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 201);
+
+        String cId = esbRestResponse.getBody().getString("Id");
+
+        methodName = "getCase";
+        String endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "GET",
+                apiRequestHeadersMap);
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 200);
+
+        methodName = "updateCase";
+        endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "PUT",
+                apiRequestHeadersMap, "esb_caseUpdate_optional.json");
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 204);
+
+        methodName = "deleteCase";
+        endpoint = getProxyServiceURLHttp(methodName) + "?caseId=" + cId;
+        esbRestResponse = sendJsonRestRequest(endpoint, "DELETE",
+                apiRequestHeadersMap);
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 204);
+    }
+
+
```

Comment:
Shall we remove these unnecessary empty lines?

Reply:
Removed